Information/Design.md

```diff
@@ -89,3 +89,320 @@
+appropriate message, and prompt for editing input again. However, if the user again, 
+provides misguided input then the _user_ command exits. However, if the user inputs valid
+input that references a specific type of item in the game. Then the Bot will find that
+item and provide a list of all fields available to edit. This way it creates no ambiguity
+on the users perspective, for what can and cannot be edited in the Monster Hunter 
+World game via the Discord bot.
+
+A privilege user is define by roles in Discord. Each Discord user account has a role 
+on a server that the user's account is a member of. Therefore, if the role is of type 
+"privileged" then the bot will allow that user account to adjust data or edit it in the 
+database of Monster Hunter World video game. 
+
+
+As with **REQ 140**, **REQ 150** is considered more of a bonus feature and therefore is 
+held off until the second phase of development. If the backend is change. Thus, currently 
+speaking this is a requirement to design hiccup that comes as an inevitable consequence 
+form the waterfall process.  
+
+
+
+examples of _user_ command usage:
+
+`-user Josh`  -- example with _name_ parameter 
+
+`-user 2`     -- example with _id_ parameter 
+
+
+
+
+------------------------------------------------------------------------------------------
+
+
+######REQ 160 Design:
+
+
+Command name: _-my victims_ 
+
+Parameters: _name_ 
+
+**Design of _user_ command**:
+
+The command _my victims_ takes in the name of a monster in the monster hunter world video
+game, as its only parameter. It installs it into a list of monsters, and then displays 
+the list to the user via discord. If the name of the monster is not a monster in the 
+Monster Hunter World game. Then the Bot prints out an appropriate message to the user. IF
+the command is entered without any name, or parameter. Then command does nothing and prints
+out nothing to the user via Discord. 
+
+When this command is entered, it will first prompt the user for the user's account name. This 
+way each user has its own saved list of monsters killed. If the user does not have a valid 
+account name or id then the monster is not saved in any list, and an appropriate message is 
+displayed to the user via discord. The user will have to authenticate themselves via their 
+password as well. This will be the same password as in the Discord server. This way random 
+people belonging to the server this Monster Hunter World bot is associated with. Cannot 
+edit the list of monsters they have killed without the knowledge of the users pass code.
+
+There are 48 total monsters in the Monster Hunter World video game. Thus the maximum size 
+of any monster list will be 48. The smallest size of course would be if there were no 
+monsters killed, thus size = 0. Being how the upper bound of a monster list is 48. 
+printing out to discord should not cause any significant undefined bandwidth issues. 
```

Comment:
The number of monsters may change, don't specify this in the design.

Reply:
Ok, I will take a look at it and try my best to adjust it in an educated way.